README.md

```diff
@@ -1,2 +1,27 @@
-# The-Async-Force-Ep-2
-Async Exercise using Interactive Client side XHR accessing the swapi
+# The-Async-Force-Ep-1
+An **async** Exercise using **client side** XHR accessing the swapi API at https://swapi.co
+
+
+
+## Excercise
+*Remember to commit after each step.*
+
+1. Make a `/js` folder and create an `app.js` file in it.
+1. In the `index.html` file, load `app.js` via script tags.
+1. Use live-reload to serve up the files to your browser.
+1. Construct your XHR requests in the `app.js` file by scraping the value from the input field.
+1. A request to the SWAPI api should be initiated when the button is clicked.
+  Information from the data that comes back from the API should be displayed in
```

Comment:
add some more goals to the readme.
The developer should start by requesting the `/people/:id` api
then, describe what should be displayed
ex:

> Display this person's name
> Display all film titles that this person appeared in
> Display all planets this person has visited